README.md

```diff
@@ -1,8 +1,13 @@
-# api3-dao-dashboard
+# api3-dao-dashboard [![ContinuousBuild](https://github.com/api3dao/api3-dao-dashboard/actions/workflows/main.yml/badge.svg?branch=main)](https://github.com/api3dao/api3-dao-dashboard/actions/workflows/main.yml)
 
 The implementation of the DAO dashboard.
 
-[![ContinuousBuild](https://github.com/api3dao/api3-dao-dashboard/actions/workflows/main.yml/badge.svg?branch=main)](https://github.com/api3dao/api3-dao-dashboard/actions/workflows/main.yml)
+We use the `main` branch to develop new features. For production code, see the
+[production branch](https://github.com/api3dao/api3-dao-dashboard/tree/production).
+
+## Development instructions
+
+You can find development instructions in a separate [dev-README](./dev-README.md).
```

Comment:
I think this is better further towards the bottom so that users see instructions for running the container first.

Reply:
I think I prefer how Emmanuel has it now, i.e. having it close to the top.

Reply:
I would argue we should cater to users first then devs, but if you both like it here then ok.